dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/ReflectionUtils.java

```diff
@@ -229,7 +230,7 @@ public static Method findGetterMethod(String fieldName, Class<?> clazz) {
         }
       }
     }
-
+    if (property.matches("(?:is|has|get)[A-Z].*")) return _findMethod(clazz, property);
```

Comment:
FYI: So the issue here is with fields and getters that do not name align, for example a field names `isDefault` and the getter being named `isDefault` (not `getIsDefault`). So after having failed to find the getter by adding a prefix we then added this fallback strategy to see if the property name itself already got a usual getter prefix and in that case we try the exact property name as getter name.